scripts/sanityCheckNuget.fsx

```diff
@@ -0,0 +1,603 @@
+#!/usr/bin/env -S dotnet fsi
+
+open System
+open System.IO
+open System.Linq
+open System.Diagnostics
+
+open System.Text
+open System.Text.RegularExpressions
+
+#r "System.Core.dll"
+open System.Xml
+#r "System.Xml.Linq.dll"
+
+open System.Xml.Linq
+open System.Xml.XPath
+
+#r "nuget: Fsdk, Version=0.6.0--date20230214-0422.git-1ea6f62"
+
+open Fsdk
+open Fsdk.Process
+
+
+#r "nuget: Microsoft.Build, Version=16.11.0"
+open Microsoft.Build.Construction
+
+let args = Misc.FsxOnlyArguments()
+
+if args.Length > 1 then
+    printf "Usage: %s [projectFolder(optional)]" __SOURCE_FILE__
+
+    Environment.Exit 1
+
+let projDir =
+    if args.IsEmpty then
+        Directory.GetCurrentDirectory()
+    elif Directory.Exists args.[0] then
+        args.[0]
+    else
+        failwithf "Directory does not exist."
```

Comment:
@Mersho please expand commit message to include the directory path in the error message